static/styles/portico/billing.css

```diff
@@ -329,5 +329,32 @@ input[name="licenses"] {
 
     textarea {
         box-sizing: border-box;
+        color: hsl(0, 0%, 33%);
+        background-color: hsl(0, 0%, 100%);
+        border-radius: 4px;
+        vertical-align: middle;
+        border: 1px solid hsl(0, 0%, 80%);
+        padding: 4px 6px;
+        margin-bottom: 10px;
+        line-height: 20px;
+
+        box-shadow: inset 0 1px 1px hsla(0, 0%, 0%, 0.075);
+        transition: border linear 0.2s, box-shadow linear 0.2s;
+
+        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
Can you post before/after screenshots for this page? In general, I'd hope we could copy less CSS into this page, and in particular I'm not sure why the textarea would need a different font than other inputs. Maybe the right idea is move this `font-family` rule to the `sponsorship-form` element, so it applies to the `input` elements as well; that'll be less duplicative.

Reply:
The bootstrap sets other input elements to this font only, that's why I have added the `font-family` for `textarea` element. We can probably change font for all inputs together to the one that is used for inputs in the app.

Reply:
Yeah, I think probably at least transferring the font-family to the form level element will save work as we remove other bootstrap things, but it's not a blocker for integrating this.